pkg/prm/backends/docker.go

```diff
@@ -19,3 +33,33 @@ func (*Docker) Exec(tool *prm.Tool, args []string) (prm.ToolExitCode, error) {
 	// TODO
 	return prm.FAILURE, nil
 }
+
+// Check to see if the Docker runtime is available:
+// if so, return true and info about Docker on this node;
+// if not, return false and the error message
+func (d *Docker) Status() prm.BackendStatus {
+	// The client does not error on creation if the background service is not running,
+	// but attempting to list the containers does.
+	dockerInfo, err := d.Client.ServerVersion(context.Background())
+	if err != nil {
+		message := fmt.Sprintf("%s", err)
+		// This is 90% likely the reason this command fails;
+		// the alternative error message is lengthy and includes
+		// references to pipes and the API which are more likely
+		// to confuse than help; so trim it to the most useful info.
+		daemonNotRunning, _ := regexp.Compile(`error during connect: This error may indicate that the docker daemon is not running.`)
+		notRunningMessage := daemonNotRunning.FindString(message)
+		if notRunningMessage != "" {
```

Comment:
Could this be simplified?

```suggestion
		daemonNotRunning := "error during connect: This error may indicate that the docker daemon is not running."
		if strings.Contains(message, daemonNotRunning) {
			message = daemonNotRunning
```